frog/frog.scrbl

```diff
@@ -984,3 +868,152 @@ In your Frog project directory, create an output directory named
+
+@defparam[current-show-tag-counts? v boolean? #:value #t]{Whether to
+show the count of posts next to each tag in the
+@secref["page-template"] variable @tt{tags/feeds}.}
+
+@defparam[current-permalink v string? #:value "/{year}/{month}/{title}.html"]{
+Pattern for blog post permalinks. An example of the Jekyll "pretty"
+style would be
+@racket["/blog/{year}/{month}/{day}/{title}/index.html"].
+
+Another available pattern is @tt{{filename}}, which is the
+@tt{this-part} portion of your post's @tt{YYYY-MM-DD-this-part.md}
+file name. This is in case you don't like Frog's encoding of your post
+title and want to specify it exactly yourself, e.g. to match a
+previous blog URI.}
+
+@defparam[current-posts-index-uri v string? #:value "/index.html"]{
+URI of the first index page for posts.}
+
+@defparam[current-index-full? v boolean? #:value #t]{Should index page
+items contain full posts -- more than just the portion above ``the
+jump'' @litchar{<!-- more -->} marker (if any)?}
+
+@defparam[current-index-newest-first? v boolean? #:value #t]{Should
+index page items be sorted newest first?}
+
+@defparam[current-feed-full? v boolean? #:value #t]{Should feed items
+contain full posts -- more than just the portion above ``the jump''
+@litchar{<!-- more -->} marker (if any)?}
+
+@defparam[current-posts-per-page n exact-positive-integer? #:value 10]{
+How many posts per page for index pages?}
+
+@defparam[current-max-feed-items n exact-positive-integer? #:value 20]{
+How many items to include in feeds? Older items in excess of this will
+not appear in the feed at all.}
+
+@defparam[current-decorate-feed-uris? v boolean? #:value #t]{Decorate
+feed URIs with Google Analytics query parameters like
+@tt{utm_source}?}
+
+@defparam[current-feed-image-bugs? v boolean? #:value #t]{Insert in
+each feed item an image bug whose URI is decorated with Google
+Analytics query parameters like @tt{utm_source}?}
+
+@defparam[current-source-dir v path-string? #:value "_src"]{
+The source directory.
+
+If you deploy to GitHub pages then it is simplest to keep this under
+the repo/project top directory.
+
+This may be an absolute or relative path. If relative, it's relative
+to the project top directory, i.e. to where @secref["config"] is
+located.}
+
+@defparam[current-output-dir v path-string? #:value "."]{
+The output directory where generated HTML and other files should go.
+
+If you deploy to e.g. GitHub pages then it is simplest to put the
+output in the repo/project top directory, which is why this defaults
+to @racket["."]. But you may change it if you prefer to copy the
+output files to their final destination.
+
+This may be an absolute or relative path. If relative, it's relative
+to the project top directory, i.e. to where @secref["config"] is
+located.}
+
+
+@section[#:tag "body-enhancers"]{Body enhancers}
+
+@defmodule[frog/enhance-body]
+
+@defproc[(syntax-highlight
+          [xs (listof xexpr/c)]
+          [#:python-executable python-executable string?]
```

Comment:
Okay. Maybe leave a note that `python-executable` gets passed to `find-executable-path`?

Reading the docs, I wasn't sure if `python-executable` needed to be the name of something in `/usr/bin`, or an absolute path, or if it could even be an alias that I set in my `.bashrc`.